misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/picking/PickLineScanScreen.jsx

```diff
@@ -162,7 +183,8 @@ export const convertScannedBarcodeToResolvedResult = ({ scannedBarcode, expected
   if (
     expectedProductNo != null &&
     parsedHUQRCode.productNo != null &&
-    String(parsedHUQRCode.productNo) !== expectedProductNo
+    String(parsedHUQRCode.productNo) !== expectedProductNo &&
+    String(parsedHUQRCode.productNo) !== expectedEAN13ProductCode
```

Comment:
i think we can remove the productNo checking because we are already doing it a bit later, onResult.

But, if we want to remove it, we would also need a playwright test using an ean13 barcode for a different product.